dev/plugins/hu.elte.txtuml.api.stdlib/.gitignore

```diff
@@ -1,4 +1,3 @@
 bin/
 target/
-src-gen/*
```

Comment:
@kovacsgabor, could you please provide some details about the compilation problem you mentioned in the commit message of e2f6d3a?

Reply:
I believe this is caused by the very same reason we need to import the stdlib project alongside the demo models. That is, we have no Maven plugin to define when and how to compile XtxtUML files during the build process. While the ultimate solution would be to create such a plugin, I agree that versioning the files generated by the IDE is enough for now.